base/uk.ac.stfc.isis.ibex.ui.journalviewer/src/uk/ac/stfc/isis/ibex/ui/journalviewer/SearchInput.java

```diff
@@ -36,18 +36,23 @@
 import org.eclipse.swt.widgets.Combo;
 import org.eclipse.swt.widgets.Composite;
 import org.eclipse.swt.widgets.Control;
-import org.eclipse.swt.widgets.DateTime;
 import org.eclipse.swt.widgets.Display;
 import org.eclipse.swt.widgets.Label;
 import org.eclipse.swt.widgets.Spinner;
 import org.eclipse.swt.widgets.Text;
 
+import uk.ac.stfc.isis.ibex.ui.journalviewer.models.JournalViewModel;
+import uk.ac.stfc.isis.ibex.ui.widgets.DateTimeWithCalendar;
+
+
 /**
  * The Class Search Control to allow searching within the log.
  */
 @SuppressWarnings("checkstyle:magicnumber")
 public class SearchInput extends Composite {
 
+    private final int runNumberIndex = 0, startTimeIndex = 2;
```

Comment:
Although it's valid syntax, it is better to keep declarations to one per line. I think checkstyle will probably complain about this (but can't check as the gui build currently won't run).

Reply:
fixed.